scm/doc/TechGuide/chap_hsd.tex

```diff
@@ -0,0 +1,212 @@
+
+\subsection{Creating Custom Data for Simulator}
+\label{section:Creating_Custom_Data_for_Simulator}
+
+Navigate to \execout{ccpp-scm/scm/etc/scripts/ccpp\_scheme\_sim}
+
+Provided with the SCM are scripts to generate data for the suite simulator using output from a previous SCM run. The first script, \execout{create\_1D\_CSSdata.py}, extracts the physics tendencies from a user specified time interval, which are used for constant forcing in the suite simulator. The other script, \execout{create\_2D\_CSSdata.py}, creates a two-dimensional forcing dataset. The suite simulator interpoaltes these forcings in time.
+
+\begin{enumerate}
+\item Run the SCM, using TWPICE case with \execout{GFS\_v17\_p8} suite.
+\begin{lstlisting}[language=bash]
+cd ccpp-scm/scm/bin
+./run_scm.py -c twpice -s SCM_GFS_v17_p8
+\end{lstlisting}
+\item Create 2D forcing data for the CSS, using SCM output from TWPICE case with \execout{GFS\_v17\_p8} suite.
+\begin{lstlisting}[language=bash]
+cd ccpp-scm/scm/etc/scripts/ccpp_scheme_sim
+./create_2D_CSSdata.py --cases twpice --suites SCM_GFS_v17_p8
+\end{lstlisting}
+\item Create constant forcing data for the CSS, using SCM output, at forecast time 3600s, from TWPICE case with \execout{GFS\_v17\_p8} suite.
+\begin{lstlisting}[language=bash]
+cd ccpp-scm/scm/etc/scripts/ccpp_scheme_sim
+./create_1D_CSSdata.py --cases twpice --suites SCM_GFS_v17_p8 --time 3600
+\end{lstlisting}
+\end{enumerate}
+
+The data file will be written to \execout{ccpp-scm/scm/etc/scripts/ccpp\_scheme\_sim/} with the following format, \execout{data\_scheme\_sim\_DIM.CASES.SUITES.nc}.
+
+\subsection{Example 1: Suite with Active Radiation}
+\label{section:Suite_with_Active_Radiation}
+
+For this example we will use the two-dimensional forcing data from \ref{section:Creating_Custom_Data_for_Simulator}. 
+
+First, we need to modify the SDF to include the CSS, \execout{ccpp\_suite\_simulator.F90} and an additional interstital scheme to couple to the GFS physics, \execout{GFS\_ccpp\_suite\_sim\_pre.F90} (See \ref{fig:CSS_SDF_ex1}).
+
+\begin{figure}[t]
+    \centering
+    \includegraphics[width=0.80\textwidth]{images/SDF_changes_for_CSS_ex1.png}
+    \caption{Changes to GFS v16 physics SDF to include CCPP suite simulator for \execout{active} radiation parameterization. All other parameterizations are replaced by simulated data.}
+    \label{fig:CSS_SDF_ex1}
+\end{figure}
+
+Next, the physics namelist needs to be configured to:
+\begin{enumerate}
+\item Add data file, \execout{suite\_sim\_file} created in \ref{section:Creating_Custom_Data_for_Simulator} to the namelist.
+\item Turn ``off'' all schemes except the radiation (see Fig. \ref{lst_css_nml})
```

Comment:
This seems to point to the wrong namelist. I think you want Listing 7.1 here

Reply:
Fig 7.1 is the tendency schematic. 7.2 and 7.3 are the SDF modifications.